src/gossip/table.zig

```diff
@@ -698,11 +698,10 @@ pub const GossipTable = struct {
         bincode.free(self.allocator, versioned_value.value.data);
     }
 
+    // Trim when over 90% of max capacity
```

Comment:
method comments need to have three - ie, '///' 

https://ziglang.org/documentation/master/#Doc-Comments